server/views/partials/layout.njk

```diff
@@ -58,4 +58,8 @@
   <script src="/assets/govukFrontendInit.js"></script>
   <script src="/assets/moj/all.js"></script>
   <script src="/assets/mojFrontendInit.js"></script>
+  {% block pageScripts %}
+     <script src="/assets/card.js"></script>
```

Comment:
This explicit reference is not needed - the `block` directive includes the code in the block in the njk template that uses it.

So you just need:

```
{% block pageScripts %}
{%  endblock %}
```